### PR TITLE
Add CLI command to remove dependencies

### PR DESCRIPTION
**Pull request by jcarpanelli**
Implemented a new cli command for unlinking a list of multiple dependencies. 

_Example usage:_

```console
foo@bar:~$ zos unlink openzeppelin-zos gnosis-safe
```

### CODE REVIEW

File: packages/cli/src/commands/unlink.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+import unlink from '../scripts/unlink'
+import push from './push'
+
+const name = 'unlink'
+const signature = `${name} [libraries...]`
+const description = 'unlinks libraries from the project. Provide a list of whitespace-separated library names'
```
Comment (facuspagnuolo) [inline packages/cli/src/commands/unlink.js]: maybe you have already thought about it, and it's not due to this PR but I noticed we are still using many terminologies for the same idea: stdlib, dependency, and library.. we should stick to only one or at least be clear with the difference between them
cc @spalladino

File: packages/cli/test/scripts/unlink.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  it('throws an error if project is a library', async function () {
+    this.packageFile.lib = true
+    const libNames = ['mock-stdlib', 'mock-stdlib-2']
+    await unlink({ libNames, packageFile: this.packageFile })
+      .should.be.rejectedWith('Libraries do not use stdlibs.')
+  })
+
+  it('throws an error if project library does not exist', async function () {
+    const libName = 'bulbasaur-lib2'
```
Comment (facuspagnuolo) [inline packages/cli/test/scripts/unlink.test.js]: 😂

File: packages/cli/test/scripts/unlink.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    await unlink({ libNames: [libName], packageFile: this.packageFile })
+    dependencies.should.not.have.property(libName)
+  })
+
+  it('unlinks multiple dependencies', async function () {
+    const { dependencies } = this.packageFile
+    const libNames = ['mock-stdlib', 'mock-stdlib-2']
+    await unlink({ libNames, packageFile: this.packageFile })
+    dependencies.should.not.have.property(...libNames)
+  })
```
Comment (facuspagnuolo) [inline packages/cli/test/scripts/unlink.test.js]: I would add a test to ensure that the dependencies you are not unlinking are still linked

File: packages/cli/src/commands/unlink.js (hunk 1 above)
Comment (jcarpanelli) [reply packages/cli/src/commands/unlink.js]: Agree. It's a bit annoying and at this point the terminology has become a bit confusing. 
+1 If you want to discuss which one we should pick and why!

**Comment by facuspagnuolo** [comment]
There are some conflicts @jcarpanelli, feel free to fix them and merge!

**Pull request merged**